cmd/pj-rehearse/main.go

```diff
@@ -161,8 +161,11 @@ func main() {
 			gracefulExit(o.noFail, "")
 		}
 	}
+
 	loggers := rehearse.Loggers{Job: logger, Debug: debugLogger.WithField(prowgithub.PrLogField, prNumber)}
-	success, err := rehearse.ExecuteJobs(changedPresubmits, prNumber, o.candidatePath, jobSpec.Refs, !o.dryRun, loggers, pjclient)
+
+	executor := rehearse.NewExecutor(changedPresubmits, prNumber, o.candidatePath, jobSpec.Refs, !o.dryRun, loggers, pjclient)
```

Comment:
Passing `!o.dryRun` (negated) does not make sense here, because it's received and saved as `dryRun`. It made sense before, because it was received as `follow`.